utils/create_dummy_models.py

```diff
@@ -264,6 +273,18 @@ def build_processor(config_class, processor_class):
             if config_class_from_processor_class != config_class:
                 processor = build_processor(config_class_from_processor_class, processor_class)
 
+    # Try to create an image processor or a feature extractor without any checkpoint
```

Comment:
Unlike tokenizers, image processor / feature extractor could be created without a checkpoint.
(This is also what has been done on `main` pipeline tests)